internal/rbd/rbd_util.go

```diff
@@ -229,6 +230,16 @@ func createImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) er
 		return fmt.Errorf("failed to create rbd image: %w", err)
 	}
 
+	if pOpts.ThickProvision {
+		err = pOpts.allocate(ctx)
+		if err != nil {
+			// nolint:errcheck // deleteImage() will log errors in
+			// case it fails, no need to log them here again
+			_ = deleteImage(ctx, pOpts, cr)
```

Comment:
can we log an error if delete fails? because if the allocation fails, the image deletion might fail but we will remove the omap entry still this can leave stale resources.

Reply:
sure. will do!

Reply:
> sure. will do!

Actually, isn't the comment above the not logging of the error sufficient?

Reply:
yeah, sound like we need to improve error logging. not to log in internal functions.